scripts/staging-sync/copy.sh

```diff
@@ -0,0 +1,37 @@
+#! /bin/bash
+#
+
+read -p "Are you sure? This *WILL* destroy any data in STA!" -n 1 -r
+echo    # (optional) move to a new line
+if [[ ! $REPLY =~ ^[Yy]$ ]]
+then
+    [[ "$0" = "$BASH_SOURCE" ]] && exit 1 || return 1 # handle exits from shell or function but don't exit interactive shell
+fi
+
+DEV=15432
+STA=15433
+PROD=15434
+
+STA_PASS=$(gcloud --project=btv-platform-sta-2 secrets versions access latest --secret="postgres_builder_password")
+PROD_PASS=$(gcloud --project=btv-platform-prod-2 secrets versions access latest --secret="postgres_builder_password")
+
+cloud_sql_proxy -verbose -instances btv-platform-dev-2:europe-west4:main-instance=tcp:localhost:$DEV -instances btv-platform-sta-2:europe-west4:main-instance=tcp:localhost:$STA -instances btv-platform-prod-2:europe-west4:main-instance=tcp:localhost:$PROD &
```

Comment:
Is there a reason you open a connection to dev as well here?

Reply:
Counting the days when we will need to copy to dev :D
But no, I'll comment it out for now.